aws/s3/secret-files/azure-secret.yaml

```diff
@@ -0,0 +1,3 @@
+azure_secret:
+  - abc7Q~defghijklmnopqrs0t123456789-_.~
+  - 61g7fT4e3ghijklmnopqrs0t123456789-_.~
```

Comment:
due to current limitations the second secret will be be reported b yCLI. Could we change the content to be more like aws above, e.g.

```
azure:
  - azure_secret1: "abc7Q~defghijklmnopqrs0t123456789-_.~"
  - azure_secret2: "61g7fT4e3ghijklmnopqrs0t123456789-_.~"
```

Reply:
Gotcha! Sure thing.

Reply:
@AnPucel could you please update `secrets.yaml` as well? thanks.

Reply:
Fixed & verified it was being picked up by the CLI.

Reply:
I'm wondering if the same applies to slack secrets. It seems like those aren't getting picked up by the CLI.